### ISSUE TITLE
Unresolvable hostname (blocked by policy) for cosmetica.cc

### ISSUE THREAD

**Issue by Bitslayn** [andrieslouw/imagesweserv#394]
Having the same issue with [https://cosmetica.cc](https://cosmetica.cc).
One such image is [https://cosmetica.cc/page/splash/Eyezah-Bottom-2.jpg](https://cosmetica.cc/page/splash/Eyezah-Bottom-2.jpg).
If you could whitelist my domain that would be great, thanks!

_Originally posted by @eyezahhhh in https://github.com/weserv/images/issues/286#issuecomment-1758881589_

**Comment by Bitslayn** [comment]
Other links which are found on https://modrinth.com/mod/cosmetica include:
https://wsrv.nl/?url=https%3A%2F%2Fcosmetica.cc%2Fpage%2Fsponsorships%2Fzap%2Fbanner-2.jpg&n=-1
https://wsrv.nl/?url=https%3A%2F%2Fcosmetica.cc%2Fpage%2Fdean-norris.png&n=-1

**Comment by kleisauke** [comment]
Duplicate of #339
